src/NuGet.Services.BasicSearch/Startup.cs

```diff
@@ -54,23 +54,25 @@ public class Startup
             // Add telemetry initializers
             TelemetryConfiguration.Active.TelemetryInitializers.Add(new MachineNameTelemetryInitializer());
 
+            // Create telemetry sink
+            _searchTelemetryClient = new SearchTelemetryClient();
+
             // Add telemetry processors
             var processorChain = TelemetryConfiguration.Active.TelemetryProcessorChainBuilder;
 
             processorChain.Use(next =>
             {
-                var responseCodeProcessor = new TelemetryResponseCodeProcessor(next);
+                var responseCodeProcessor = new RequestTelemetryProcessor(next);
```

Comment:
The type changed but the variable name is the same. Maybe change it to align with the new type?